packages/cfd/src/Containers/compare-accounts-modal.tsx

```diff
@@ -35,6 +36,47 @@ type TCompareAccountsModalProps = TCompareAccountsReusedProps & {
     openDerivRealAccountNeededModal: () => void;
 };
 
+type TDxtradeCompareAccountContent = TCompareAccountsReusedProps & {
+    is_demo_tab: boolean;
+    show_eu_related: boolean;
+    residence: string;
+    is_eu: boolean;
+};
+
+// TODO: Remove this component and use one component for both when real released.
+const DxtradeCompareAccountContent = ({
+    is_demo_tab,
+    is_logged_in,
+    landing_companies,
+    platform,
+    show_eu_related,
+    residence,
+    is_eu,
+    is_uk,
+}: TDxtradeCompareAccountContent) => {
+    return is_demo_tab ? (
```

Comment:
@hamid-deriv Ternary operator is not recommended if your condition can’t fit on a single line, Let’s use `return early` instead for better readability 🙏🏻 

```js
if (is_demo_tab) {
  return <CfdDxtradeCompareContent />
}

return <ModalContent />
```